cogs/vip.py

```diff
@@ -18,6 +18,46 @@
 class VIP(Cog):
     cog_command_error = cog_error_handler
 
+    #check if vip is still valid
+	#this is probably going to need to be threaded?
+    @Cog.listener()
+    async def on_message(self, message):
```

Comment:
imo we should do this via a cronjob instead of this way.

Basically it would work like this:
- Use API to get discord id's of users who are "vip role"
- Does a single db query to get the users who don't have VIP any longer by discord id

```
discord_ids = get_vip_users()
query = f"select user_id from xf_user_field_value where field_id = discord_user_id and field_value in ({discord_ids})"

result = await database.fetch_one(query=query)

for user in result:
    # remove user from discord vip role
```